cloud/services/publicips/publicips.go

```diff
@@ -54,8 +53,10 @@ func (s *Service) Reconcile(ctx context.Context) error {
 		if err != nil {
 			return errors.Wrap(err, "cannot create public IP")
 		}
-		klog.V(2).Infof("successfully created public IP %s", ip.Name)
```

Comment:
I see a few more klog calls in the file 

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/747/files#diff-55ae330727dcaa61a46e9aca5c270946L75

Reply:
done